proto/felixbackend.proto

```diff
@@ -272,6 +272,13 @@ message ServiceAccountMatch {
 
 message HTTPMatch {
   repeated string methods = 1;
+  message path_match_types{
```

Comment:
Message names are initial caps, and I think we should drop `types`.  Yes, the types are `exact` and `prefix`, but this message encodes both the type and value, so it should just be called `PathMatch`